src/test/test_marshal.cpp

```diff
@@ -133,22 +133,32 @@ TEST_CASE("disasm_marshal", "[disasm][marshal]") {
             }
         }
     }
-
-    SECTION("Un") {
-        auto ops = {
-            Un::Op::BE16,
-            Un::Op::BE32,
-            Un::Op::BE64,
-            Un::Op::LE16,
-            Un::Op::LE32,
-            Un::Op::LE64,
-            Un::Op::NEG,
-            Un::Op::SWAP16,
-            Un::Op::SWAP32,
-            Un::Op::SWAP64
-        };
-        for (auto op : ops)
-            compare_marshal_unmarshal(Un{.op = op, .dst = Reg{1}, .is64 = true});
+    SECTION("Neg") {
+        compare_marshal_unmarshal(Un{.op = Un::Op::NEG, .dst = Reg{1}, .is64 = false});
+    }
+    SECTION("Endian") {
+        // FIX: `.is64` field is unused; the difference in value is accidental.
```

Comment:
It's not accidental per se, it matches what the [ISA spec says](https://www.ietf.org/archive/id/draft-ietf-bpf-isa-00.html#byte-swap-instructions) (i.e., it matches BPF_ALU vs BPF_ALU64).

Reply:
Accidental in the slightly "abstract syntax" that PREVAIL use (or does it have a meaning I miss?)